catroid/src/androidTest/java/org/catrobat/catroid/test/ProjectManagerTest.java

```diff
@@ -172,4 +174,46 @@ public void testLoadProjectWithInvalidNestingBrickReferences() throws IOExceptio
 
 		TestUtils.deleteProjects(PROJECT_NAME_NESTING_BRICKS);
 	}
+
+	@Test
+	public void testMergeProjectWithNoConflicts() throws IOException {
+		Project project1 = TestUtils.createProjectWithLanguageVersion(CURRENT_CATROBAT_LANGUAGE_VERSION,
+				"testProject1");
+		Project project2 = TestUtils.createProjectWithLanguageVersion(CURRENT_CATROBAT_LANGUAGE_VERSION,
+				"testProject2");
+		Scene sceneOne = new Scene("test1", project1);
```

Comment:
stick to one naming convention, either everything is One, Two,... OR 1,2,...

Reply:
Resolved